src/js/entities-service/entities/teams.js

```diff
@@ -7,6 +8,17 @@ const TYPE = 'teams';
 const _Model = BaseModel.extend({
   type: TYPE,
   urlRoot: '/api/teams',
+  getAssignableClinicians() {
```

Comment:
I think this function should be:
```js
getAssignableClinicians() {
  const clinicians = Radio.request('entities', 'clinicians:collection', this.get('_clinicians'));

  clinicians.filterAssignable();

  return clinicians;
},
```
where filterAssignable is something like this on the clinician:
```js
const assignable = this.filter(clinician => {
  return clinician.isActive() && clinician.get('enabled') && clinician.can('work:own');
});

this.reset(assignable);
return this;
```
so that how we determine what an assignable clinician is, is in one location... so on workspace we'd use `filterAssignable` as well